src/storage-client/src/types/errors.proto

```diff
@@ -45,10 +46,15 @@ message ProtoUpsertValueError {
     mz_repr.row.ProtoRow for_key = 2;
 }
 
+message ProtoUpsertNullKeyError {
+    /* mz_expr.id.ProtoPartitionId partition_id = 1; */
```

Comment:
are we sure we don't want this now? this is basically our only chance...

Reply:
not at all sure! let's figure it out next week